src/main/java/frc/robot/Constants.java

```diff
@@ -238,4 +242,22 @@ public static class GamePieceManipulatorConstants {
 
   }
 
+  /*
```

Comment:
this should be /** for a javadoc comment. /* starts a multiline comment, /** starts a javadoc comment.